src/modules/client/client.controller.ts

```diff
@@ -0,0 +1,102 @@
+import { 
+  Body,
+  Controller, 
+  Delete, 
+  Get, 
+  Param, 
+  ParseUUIDPipe, 
+  Post, 
+  Put, 
+  Query, 
+  Req, 
+  UseGuards
+} from '@nestjs/common';
+import {
+  ApiBadRequestResponse,
+  ApiBearerAuth,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
+import { Client } from '../../../entities/Client.entity';
+import { Request } from 'express';
+import { ClientService } from './client.service';
+import { DeleteResult } from 'typeorm';
+import { ParseOrderByPipeClients, ParseOrderPipeClients } from './client.pipes';
+import { PageResponse } from 'src/constants/PageResponse';
+
+@Controller('/client')
+@ApiTags('Clients')
+@ApiBearerAuth()
+@UseGuards(JwtAuthGuard)
+@ApiUnauthorizedResponse({ description: 'Unauthorized User' })
+@ApiBadRequestResponse({ description: 'Bad Request' })
+export class ClientController {
+  constructor(private readonly clientService: ClientService) {}
+
+  @Get('/:clientId')
+  @ApiOperation({summary: 'Retrieve details about an existing Client'})
+  @ApiOkResponse({ description: 'All info of a User', type: Client })
+  async getClientInfoById(
+    @Param('clientId', ParseUUIDPipe) clientId: string
```

Comment:
@fernandogcabal please include as parte of the params to get the client information the companyId, the same way we do it in here:
https://github.com/matheo97/restaurant/blob/a519b14f6a161343a65655b7f0aa6c72f656a764/src/modules/item/item.controller.ts#L58-L66